.env.example

```diff
@@ -10,3 +10,5 @@ RINKEBY_PROVIDER=https://eth-rinkeby.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 KOVAN_PROVIDER=https://eth-kovan.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 GOERLI_PROVIDER=https://eth-goerli.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 
+ETHERSCAN_API_KEY=""
```

Comment:
Does this also require a new object in the hardhat config file?

https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html#usage

Reply:
It does, our hardhat config file already had that Etherscan object.